src/bundle/Resources/views/themes/admin/ui/menu/user.html.twig

```diff
@@ -5,25 +5,45 @@
     {% set user_image = user.content.thumbnail %}
 
     <div class="ibexa-header-user-menu">
-        <img 
-            class="ibexa-header-user-menu__image" 
-            src="{{ user_image.resource }}" 
-            alt="{{ user.name }}" 
+        <div
+            class="ibexa-header-user-menu__notifications-toggler"
             data-bs-toggle="modal"
             data-bs-target="#view-notifications"
-        />
-        <span
-            class="ibexa-header-user-menu__notice-dot ibexa-header-user-menu__notice-dot--no-notice"
-            data-bs-toggle="modal"
-            data-bs-target="#view-notifications"
-        ></span>
-        <button class="ibexa-header-user-menu__name">{{ user.name }}</button>
+        >
+            <svg class="ibexa-icon ibexa-icon--small ibexa-icon--light">
+                <use xlink:href="{{ ibexa_icon_path('bell') }}"></use>
+            </svg>
+            <span class="ibexa-header-user-menu__notice-dot ibexa-header-user-menu__notice-dot--no-notice"></span>
+        </div>
+        <button class="ibexa-header-user-menu__toggler">
+            {% if user_image.resource is same as("/bundles/ibexaicons/img/all-icons.svg#user") %}
```

Comment:
```suggestion
            {% if user_image.resource is same as(ibexa_icon_path('user')) %}
```
?

Reply:
this is a temporary solution, Sławek should prepare a proper one next week. https://issues.ibexa.co/browse/IBX-1572

Reply:
Should we add TODO?

Reply:
Ticket is better than TODO